src/features/duplicates/utils/sortValuesByFrequency.ts

```diff
@@ -0,0 +1,34 @@
+export default function sortValuesByFrequency(values: string[]) {
+  const frequency: Record<string | number, number> = {};
+
+  values.forEach((value) => {
+    if (!frequency[value]) {
+      frequency[value] = 0;
+    }
+    frequency[value] = frequency[value] + 1;
+  });
+
+  const uniqueValues = [...new Set(values)];
+
+  const sortedByFrequency = uniqueValues.sort((value1, value2) => {
+    const a = value1 || '';
+    const b = value2 || '';
+
+    if (frequency[a] > frequency[b]) {
+      return -1;
+    } else if (frequency[b] > frequency[a]) {
+      return 1;
+    } else {
+      return 0;
+    }
```

Comment:
I think this whole function is great. My only comment is this. I believe these if/else branches could have been replaced by just `return frequency[b] - frequency[a]`. No need to change it now. Just pointing it out for the future.